### ISSUE TITLE
inputRange must be monotonically increasing

### ISSUE THREAD

**Issue by EdwardNelson** [wix/react-native-calendars#87]
I receive "inputRange must be monotonically increasing 0,-104" error when using wix react native navigation tabs with the Agenda component. However, if I put the agenda component in the screen assigned to the first tab I do not get this error.

**Comment by EdwardNelson** [comment]
It is late here, but as a quick starting point as to where the problem might be I did this:

Change 

https://github.com/wix/react-native-calendars/blob/master/src/agenda/index.js#L88-L90

to 

```
initialScrollPadPosition() {
    if(this.viewHeight > HEADER_HEIGHT) {
      return this.viewHeight - HEADER_HEIGHT;
    }

    return 0
  }
```

**Comment by tautvilas** [comment]
@overengineered if this is hard to fix maybe we can just make draggable agenda optional for now?

**Comment by overengineered** [comment]
I don't think its hard, @EdwardNelson probably provided correct solution, it might be incomplete though.

**Comment by EdwardNelson** [comment]
@overengineered Is there anything I can do to speed up a resolution to this?

Is it a case of doing the same check in these places?

https://github.com/wix/react-native-calendars/blob/master/src/agenda/index.js#L88-L90
https://github.com/wix/react-native-calendars/blob/master/src/agenda/index.js#L247

**Comment by overengineered** [comment]
I will look into this problem this evening, I expect it to be fixed tomorrow

**Comment by tautvilas** [comment]
fixed in 1.5.1

**Comment by Johncy1997** [comment]
<Carousel
                
            autoplay
            autoplayTimeout={5000}
            loop
            index={0}
            showsPageIndicator={ false }
             pageSize={BannerWidth}
            height={250}
            itemWidth={sliderWidth}
      itemHeight={itemHeight}
        >
        { 
          
          this.props.data.banner.map((image, index) => {
            if(image !== null){
          console.log("carousel started to render"+image)
          return(
            <View key={index}> 
          {                          console.log("The image url is "+image.BannerUrl)
           }
              <Image source={{uri: image.BannerUrl}} style={{flex:1,width: BannerWidth, height: BannerHeight,resizeMode:'contain'}}/>
            </View>
          )}
        })}
        </Carousel>


![screenshot_2018-03-08-10-27-07](https://user-images.githubusercontent.com/25324938/37137385-7bebfc32-22cb-11e8-9fd4-26d24ab7a817.png)


this.props.data.banner=[{BannerUrl:'..',..},{...},{....}]
I want to render the images in a BannerUrl to this Carousel.I can't fix it from yesterday and also can't understand the problem here.Plz someone tell me.Thanks
